GA/post.py

```
@@ -0,0 +1,16 @@
+from pymoo.core.mixed import MixedVariableGA
```

Ruff (0.8.2)

`pymoo.core.mixed.MixedVariableGA` imported but unused

Remove unused import: `pymoo.core.mixed.MixedVariableGA`

(F401)